apps/testclient/management/commands/create_test_users_and_applications_batch.py

```diff
@@ -94,33 +116,36 @@ def create_dev_users_apps_and_bene_crosswalks(group, bene_count, dev_count, app_
                         "email": fn + '.' + ln + "@xyz.net",
                     }
                     slsx_client = OAuth2ConfigSLSx(args)
```

Comment:
What's the Oauth config for?

Reply:
It's created so that it can be input to create_beneficiary_record. This mimics what happens during an actual authorization flow.